src/IdentityBar/index.js

```diff
@@ -1,30 +1,30 @@
 import React from 'react'
 
-export const BrandBar = (props) => {
+export const IdentityBar = (props) => {
 
   // Defaults.
   const defaultClasses = {
```

Comment:
I'm not a fan of what I did here with the class arrays. This takes away too much from how clear and easy it is with working in these templates. I want to try another approach where I separate out the variants into separate templates.

Reply:
Not to take away from these updates but I am going to try a refactor of this approach that we might be able to discuss on friday.

Reply:
Sounds great! Thanks.